index.html

```diff
@@ -256,6 +261,26 @@ <h3>The <dfn><code>VisualViewport</code></dfn> interface</h3>
                 Although it is referred to as the pinch-zoom scaling factor, it can be affected through means other than pinch-zooming. e.g. When the UA centers and zooms in a focused edit box.
               </div>
             </dd>
+            <dt><dfn>segments</dfn></dt>
+            <dd>
+              <p>
+                Returns an array of DOMRects that represent the dimensions of each existing viewport segment. 
+              </p>
+              <p>
+                A viewport segment represents the region of the visual viewport that resides on a separate display or logical display region. 
+                When the visual viewport spans across some number of physical features of a device, these features in turn convey a logical separation of the space available for content.
+              </p>
+              <p>
+                <code>segements</code> is null when there is only a single viewport segment. Returns null when called from within an iframe context.
+              </p>
+            </dd>
+            <div class="note">
+              Based on the data returned for each viewport segment, developers will be able to infer the number of hinges available as well as the hinge orientation.
+              The browser window can be moved/resized such that the number of segments and/or their dimensions change. In these cases, resize events will fire at which point authors should re-query this property, as the returned FrozenArray is just a snapshot of the current state. 
+            </div>
+            <div class="note">
+              The viewport segments code is currently in development and considered experimental. 
```

Comment:
nit: should read the "segments property" or something similar instead of referencing 'code'